### ISSUE TITLE
{{ noparse }} error in Antlers runtime

### ISSUE THREAD

**Issue by ryanmitchell** [statamic/cms#5739]
### Bug description

When using the no parse tag in a template with instant search:

```
{{ noparse }}
        instantsearch.widgets.hits({
            container: '#hits',
            templates: {
              item: `
                <div>
                  <div class="hit-name">
                    <a href="{{url}}">{{#helpers.highlight}}{ "attribute": "name" }{{/helpers.highlight}}</a>
                    <p>{{ description }}</p>
                    <p>Price: \${{ price }}</p>
                  </div>
                </div>
              `,
              empty: 'No results for <q>{{ query }}</q>',
            }
        }),
{{ /noparse }}
```

I get the error: `[ANTLR_011]: Unexpected end of input while parsing Antlers comment region.`

it seems to trip up over `{{#helpers.highlight}}` as when I remove that line it works as expected.

### How to reproduce

See code example above.

### Versions

3.3.3 and antlers: runtime

### Installation

Existing Laravel app

### Antlers Parser

runtime (new)

**Comment by duncanmcclean** [comment]
Don't suppose you've checked if this is fixed in the latest release (v3.3.4)?

**Comment by jasonvarga** [comment]
It's broken in 3.3.4 too 👍
